contracts/script/deploy/verifiable/DeployVerifiable.s.sol

```diff
@@ -116,24 +95,49 @@ contract DeployVerifiable is Script {
         // Transfer ownership of proxy admin to deployment initializer
         proxyAdmin.transferOwnership(address(deploymentInitializer));
 
+        _logs();
+
         vm.stopBroadcast();
 
-        _doTests(configData);
+        _doTests();
+    }
+
+    /// @dev override this if you don't want to use the environment to get the config path
+    function _initConfig() internal virtual {
+        cfg = vm.readFile(vm.envString("DEPLOY_CONFIG_PATH"));
     }
 
-    function _doTests(string memory cfg) internal {
-        vm.startPrank(initialOwner);
-        CalldataInitParams memory params = CalldataInitParamsLib.getCalldataInitParams(cfg);
+    function _logs() internal virtual {
+        console2.log("Deployment addresses:");
```

Comment:
just to learn - how does `console2` differ from just regular `console`?

Reply:
From the foundry [doc](https://book.getfoundry.sh/reference/forge-std/),

"console2.sol contains patches to console.sol that allow Forge to decode traces for calls to the console, but it is not compatible with Hardhat."